### PR TITLE
channel list output is confusing #755

### PR DESCRIPTION
**Pull request by parsharma**
Issue: https://github.com/wildfly-extras/prospero/issues/755

### CODE REVIEW

File: prospero-cli/src/main/java/org/wildfly/prospero/cli/commands/ChannelCommand.java [hunk 1]
```diff
@@ -53,6 +53,9 @@ public static class ChannelListCommand extends AbstractCommand {
         @CommandLine.Option(names = CliConstants.DIR)
         Optional<Path> directory;
 
+        @CommandLine.Option(names = CliConstants.FULL)
```
Comment (spyrkob) [inline prospero-cli/src/main/java/org/wildfly/prospero/cli/commands/ChannelCommand.java:56]: New options need to be described in the `UsageMessages.properties`

File: prospero-cli/src/test/java/org/wildfly/prospero/cli/commands/ChannelCommandTest.java [hunk 2]
```diff
@@ -191,7 +191,33 @@ public void testList() {
         int exitCode = commandLine.execute(CliConstants.Commands.CHANNEL, CliConstants.Commands.LIST,
                 CliConstants.DIR, dir.toString());
         Assert.assertEquals(ReturnCodes.SUCCESS, exitCode);
-        Assert.assertEquals(3, getStandardOutput().lines().filter(l->l.contains("manifest")).count());
+        Assert.assertEquals(3, getStandardOutput().lines()
```
Comment (spyrkob) [inline prospero-cli/src/test/java/org/wildfly/prospero/cli/commands/ChannelCommandTest.java:194]: Use `Assertions.assertThat` and compare it to actual expected lines ("test1 g:a", "test2 g:a:v", "test3 file:/a:b")

File: prospero-cli/src/test/java/org/wildfly/prospero/cli/commands/ChannelCommandTest.java [hunk 3]
```diff
@@ -191,7 +191,33 @@ public void testList() {
...
+                .count());
+    }
+
+    @Test
+    public void testFullList() {
+        int exitCode = commandLine.execute(CliConstants.Commands.CHANNEL, CliConstants.Commands.LIST, CliConstants.FULL,
+                CliConstants.DIR, dir.toString());
+        Assert.assertEquals(ReturnCodes.SUCCESS, exitCode);
+        String output = getStandardOutput();
+        Assert.assertTrue("Expected 'schemaVersion' in the output",
```
Comment (spyrkob) [inline prospero-cli/src/test/java/org/wildfly/prospero/cli/commands/ChannelCommandTest.java:205]: The test doesn't actually verify that the correct values are printed.  You should be able to get the output, convert it back to Channel objects and compare with the input. 
Or simply compare the output with serialized input.

**Review by spyrkob — CHANGES_REQUESTED** [review]

**Comment by parsharma** [comment]
@spyrkob, I have updated the code according to the review comments. Please have a look and let me know if more changes are needed.
Thanks

**Pull request merged**